diff --git a/node_modules/jsdom/lib/jsdom/living/events/InputEvent-impl.js b/node_modules/jsdom/lib/jsdom/living/events/InputEvent-impl.js
index 6f6dd48..89698df 100644
--- a/node_modules/jsdom/lib/jsdom/living/events/InputEvent-impl.js
+++ b/node_modules/jsdom/lib/jsdom/living/events/InputEvent-impl.js
@@ -3,7 +3,21 @@ const UIEventImpl = require("./UIEvent-impl").implementation;
 const InputEventInit = require("../generated/InputEventInit");
 
 // https://w3c.github.io/uievents/#interface-inputevent
-class InputEventImpl extends UIEventImpl { }
+class InputEventImpl extends UIEventImpl {
+    initInputEvent(type, bubbles, cancelable, data, isComposing) {
+      if (this._dispatchFlag) {
+        return;
+      }
+
+      this.initUIEvent(type, bubbles, cancelable);
+      this.data = data;
+      this.isComposing = isComposing;
+    }
+
+    getTargetRanges() {
+      return []
+    }
+}
 InputEventImpl.defaultInit = InputEventInit.convert(undefined);
 
 module.exports = {
diff --git a/node_modules/jsdom/lib/jsdom/living/generated/InputEvent.js b/node_modules/jsdom/lib/jsdom/living/generated/InputEvent.js
index 3dd61df..5883311 100644
--- a/node_modules/jsdom/lib/jsdom/living/generated/InputEvent.js
+++ b/node_modules/jsdom/lib/jsdom/living/generated/InputEvent.js
@@ -113,6 +113,15 @@ exports.install = (globalObject, globalNames) => {
       return exports.setup(Object.create(new.target.prototype), globalObject, args);
     }
 
+    getTargetRanges() {
+      const esValue = this !== null && this !== undefined ? this : globalObject;
+      if (!exports.is(esValue)) {
+        throw new TypeError("'getTargetRanges' called on an object that is not a valid instance of InputEvent.");
+      }
+
+      return utils.tryWrapperForImpl(esValue[implSymbol].getTargetRanges());
+    }
+
     get data() {
       const esValue = this !== null && this !== undefined ? this : globalObject;
 
@@ -142,11 +151,23 @@ exports.install = (globalObject, globalNames) => {
 
       return esValue[implSymbol]["inputType"];
     }
+
+    get dataTransfer() {
+      const esValue = this !== null && this !== undefined ? this : globalObject;
+
+      if (!exports.is(esValue)) {
+        throw new TypeError("'get dataTransfer' called on an object that is not a valid instance of InputEvent.");
+      }
+
+      return utils.tryWrapperForImpl(esValue[implSymbol]["dataTransfer"]);
+    }
   }
   Object.defineProperties(InputEvent.prototype, {
+    getTargetRanges: { enumerable: true },
     data: { enumerable: true },
     isComposing: { enumerable: true },
     inputType: { enumerable: true },
+    dataTransfer: { enumerable: true },
     [Symbol.toStringTag]: { value: "InputEvent", configurable: true }
   });
   if (globalObject[ctorRegistrySymbol] === undefined) {
diff --git a/node_modules/jsdom/lib/jsdom/living/generated/InputEventInit.js b/node_modules/jsdom/lib/jsdom/living/generated/InputEventInit.js
index a41cfa2..266b377 100644
--- a/node_modules/jsdom/lib/jsdom/living/generated/InputEventInit.js
+++ b/node_modules/jsdom/lib/jsdom/living/generated/InputEventInit.js
@@ -3,6 +3,7 @@
 const conversions = require("webidl-conversions");
 const utils = require("./utils.js");
 
+const StaticRange = require("./StaticRange.js");
 const UIEventInit = require("./UIEventInit.js");
 
 exports._convertInherit = (obj, ret, { context = "The provided value" } = {}) => {
@@ -23,6 +24,21 @@ exports._convertInherit = (obj, ret, { context = "The provided value" } = {}) =>
     }
   }
 
+  {
+    const key = "dataTransfer";
+    let value = obj === undefined || obj === null ? undefined : obj[key];
+    if (value !== undefined) {
+      if (value === null || value === undefined) {
+        value = null;
+      } else {
+        value = utils.tryImplForWrapper(value);
+      }
+      ret[key] = value;
+    } else {
+      ret[key] = null;
+    }
+  }
+
   {
     const key = "inputType";
     let value = obj === undefined || obj === null ? undefined : obj[key];
@@ -46,6 +62,31 @@ exports._convertInherit = (obj, ret, { context = "The provided value" } = {}) =>
       ret[key] = false;
     }
   }
+
+  {
+    const key = "targetRanges";
+    let value = obj === undefined || obj === null ? undefined : obj[key];
+    if (value !== undefined) {
+      if (!utils.isObject(value)) {
+        throw new TypeError(context + " has member 'targetRanges' that" + " is not an iterable object.");
+      } else {
+        const V = [];
+        const tmp = value;
+        for (let nextItem of tmp) {
+          nextItem = StaticRange.convert(nextItem, {
+            context: context + " has member 'targetRanges' that" + "'s element"
+          });
+
+          V.push(nextItem);
+        }
+        value = V;
+      }
+
+      ret[key] = value;
+    } else {
+      ret[key] = [];
+    }
+  }
 };
 
 exports.convert = function convert(obj, { context = "The provided value" } = {}) {
