diff --git a/node_modules/slate-hyperscript/dist/index.js b/node_modules/slate-hyperscript/dist/index.js
index 6e43e54..de9b8c5 100644
--- a/node_modules/slate-hyperscript/dist/index.js
+++ b/node_modules/slate-hyperscript/dist/index.js
@@ -699,10 +699,22 @@ var createFactory = function createFactory(creators) {
       attributes = {};
     }

+        // We are in an unejected CRA that uses its Babel config for Jest. The plugin
+    // https://babeljs.io/docs/en/babel-plugin-transform-react-jsx-source.html
+    // adds sourcemap information when parsing hyperscript (jsx) that we want to omit so
+    // when we assert we aren't asserting on the sourcemap stuff.
+    // Only do this for test environment in case we end up using Hyperscript in application
+    // code.
+    const newAttributes = Object.assign(attributes, {})
+    if (process.env.NODE_ENV === 'test') {
+      delete newAttributes.__source
+      delete newAttributes.__self
+    }
+
     children = children.filter(function (child) {
       return Boolean(child);
     }).flat();
-    var ret = creator(tagName, attributes, children);
+    var ret = creator(tagName, newAttributes, children);
     return ret;
   };

@@ -741,6 +753,14 @@ var normalizeElements = function normalizeElements(elements) {

 var jsx = createHyperscript();

+// Slate doesn't export these, but we'll want them eventually.
+exports.createAnchor = createAnchor;
+exports.createCursor = createCursor;
+exports.createElement = createElement;
+exports.createFocus = createFocus;
+exports.createFragment = createFragment;
+exports.createSelection = createSelection;
+exports.createText = createText;
 exports.createEditor = createEditor;
 exports.createHyperscript = createHyperscript;
 exports.jsx = jsx;
