diff --git a/src/Parser.js b/src/Parser.js
index 36ec34e3c00a672c03f8476eecc5ef2273445727..ef74e8aafe0fa0e6b6bff1d8dce8a1c64ebb65fa 100644
--- a/src/Parser.js
+++ b/src/Parser.js
@@ -1,6 +1,6 @@
 import {COMMENT, RULESET, DECLARATION} from './Enum.js'
 import {abs, charat, trim, from, sizeof, strlen, substr, append, replace, indexof} from './Utility.js'
-import {node, char, prev, next, peek, token, caret, alloc, dealloc, delimit, whitespace, escaping, identifier, commenter} from './Tokenizer.js'
+import {node, getChar, prev, next, peek, token, caret, alloc, dealloc, delimit, whitespace, escaping, identifier, commenter} from './Tokenizer.js'
 
 /**
  * @param {string} value
@@ -186,7 +186,7 @@ export function ruleset (value, root, parent, index, offset, rules, points, type
  * @return {object}
  */
 export function comment (value, root, parent, siblings) {
-	return node(value, root, parent, COMMENT, from(char()), substr(value, 2, -2), 0, siblings)
+	return node(value, root, parent, COMMENT, from(getChar()), substr(value, 2, -2), 0, siblings)
 }
 
 /**
diff --git a/src/Tokenizer.js b/src/Tokenizer.js
index 201fb51bf8fc603d358f92787e26cdc0ece210ab..94229b03c7791ec02f0aacfcc1c6b06432734763 100644
--- a/src/Tokenizer.js
+++ b/src/Tokenizer.js
@@ -43,7 +43,7 @@ export function lift (root) {
 /**
  * @return {number}
  */
-export function char () {
+export function getChar () {
 	return character
 }
 
